app/Http/Controllers/BookController.php

```
@@ -65,4 +65,36 @@ public function claimBook(int $id, Request $request)
             }
         }
     }
+
+    public function returnBook(int $id, Request $request)
+    {
+
+        $bookToUpdate = Book::find($id);
+
+        if ($bookToUpdate) {
+            if ($bookToUpdate->claimed == 0) {
+                return response()->json([
+                    'message' => "Book $id is not currently claimed",
+                ], 400);
+            } elseif ($bookToUpdate->claimed == 1) {
+
+                $request->validate([
+                    'email' => 'string|email|max:255|required',
```

You don't actually do anything with this email - you need to make sure that it matches the email of the current claimer of the book - if it doesn't you should return a specific error response (https://github.com/iO-Academy/book-swap-api-docs/blob/main/README.md#return-book)

**Code:** 400 BAD REQUEST   
**Content:** `{"message": "Book 10 is not currently claimed"}`